dali/base/dafdesc.cpp

```diff
@@ -3653,6 +3654,36 @@ class CStoragePlaneInfo : public CInterfaceOf<IStoragePlane>
     virtual const char * querySingleHost() const override { return xml->queryProp("@host"); }   // MORE: Likely to be changed to resolve hosts
     virtual unsigned numDefaultSprayParts() const override { return xml->getPropInt("@defaultSprayParts", 1); }
     virtual bool queryDirPerPart() const override { return xml->getPropBool("@subDirPerFilePart", isContainerized()); } // default to dir. per part in containerized mode
+    virtual bool hasStorageApi() const override { return xml->hasProp("storageapi"); }
+    virtual StorageType getStorageType() const override
+    {
+        const char * storageTypeStr = xml->queryProp("storageapi/@type");
+        if (streq(storageTypeStr, "azurefile"))
+            return StorageType::StorageTypeAzureFile;
+        else if (streq(storageTypeStr, "azureblob"))
+            return StorageType::StorageTypeAzureBlob;
+        return StorageType::StorageTypeUnknown;
+    }
+    virtual const char * queryStorageApiAccount() const override
+    {
+        const char * storageApiAccount = xml->queryProp("storageapi/@account");
+        if (isEmptyString(storageApiAccount))
+            storageApiAccount = getenv("AZURE_ACCOUNT_NAME");
```

Comment:
Minor: Environment variable shouldn't really have azure in if this is equally applicable to aws.  Is it just for testing?  Should it be required.

Reply:
I've removed the use of AZURE_ACCOUNT_NAME env var .